tracee-ebpf/tracee/vmlinux/x86_64/vmlinux-arch.h

```diff
@@ -0,0 +1,65 @@
+#ifndef __VMLINUX_ARCH_H__
+#define __VMLINUX_ARCH_H__
+
+#include <vmlinux-core.h>
+#include <vmlinux-flavored.h>
+
+#ifndef BPF_NO_PRESERVE_ACCESS_INDEX
```

Comment:
Do we need this ifndef? Can it ever be that this macro will be defined? If so, where?

Reply:
https://patchwork.ozlabs.org/project/netdev/patch/20200113073143.1779940-5-andriin@fb.com/ .. based on that git log I dont think there will be any place that BPF_NO_PRESERVE_ACCESS_INDEX will be set unless explicitly given. I'm removing the ifndef.